### DISCUSSION TITLE
Problem verifying application for github student pack

### DISCUSSION THREAD

**Discussion by Wojsonexe** [community/community#77784] [GitHub Education]
### Hi everyone,

Hi I want to apply for a package and I verify myself with a Polish school ID card but the system around the clock does not allow me to approve the error below: 
Please correct your GitHub billing information with your name exactly as it appears on your academic affiliation document. If you have only one legal name, please enter it in both the first and last name fields.
Correct your GitHub billing information by entering your first name exactly as it appears on your academic affiliation document. Do not use a variation of your first name or nickname. 

In my name I have Polish letters is it possible that because of this it does not allow me to go further ?

**Comment by dakshithadissanayaka** [comment]
Yes, it is possible that the system is not recognizing the Polish letters in your name, causing the error message to appear. GitHub's verification process may be expecting specific formatting for names, and the Polish characters might be causing an issue with the system's parsing.

To resolve this issue, you can try the following:

1. **Check for Alternate Name Options:** Look for any options to enter your name in a different format. Some verification systems may allow you to input your name in a more relaxed format, such as allowing accented characters or alternative spellings.

2. **Contact GitHub Support:** If there are no alternative name options available, reach out to GitHub support directly. They can manually review your application and verify your identity based on your provided documentation. Explain that you are encountering issues with the verification process due to Polish characters in your name.

3. **Consider Using a Different Verification Method:** If GitHub support is unable to resolve the issue, consider using a different verification method if available. Some verification processes may allow you to use alternative documents, such as transcripts or enrollment documents, which might not have the same character limitations.

Remember to provide clear and concise communication to GitHub support when explaining the issue. Attach a copy of your school ID card and highlight the Polish characters that might be causing the problem. This will help them understand the situation and assist you promptly.

**Comment by Wojsonexe** [comment]
Okay I will try to translate my name into English. I have no other option than my school ID card
